src/router-holofuel.js

```diff
@@ -1,27 +1,18 @@
-import React, { useContext } from 'react'
+import React from 'react'
```

Comment:
think this file should be called `HFRouter`. as a convention I think we should name files that default export a react component after that component.

I also think it would be great if this file were in a `holofuel` directory, to maintain the convention that all files for the holofuel app are in subdirectory called `holofuel`. Maybe just at the top level,

so we'd have `src/HPAdminRouter.js` and `src/holofuel/HFRouter.js`

Reply:
Do we really want to go with separate directory just for this file? @robbiecarlton